util/common/src/test/java/org/hibernate/search/util/common/jar/impl/CodeSourceTest.java

```diff
@@ -263,7 +268,7 @@ public void jar_jarScheme_specialCharacter() throws Exception {
 		);
 
 		URL fileURL = jarPath.toUri().toURL();
```

Comment:
Might as well use an URI here, and `fileURI.toString()` instead of `fileURL.toExternalForm()` below?